src/middleware/sentry.js

```diff
@@ -3,7 +3,11 @@ const { sha } = require('../helper/version');
 const { version } = require('../../package.json');
 const { deepObject } = require('../utils');
 const {
-	SENTRY_DSN, SC_DOMAIN, NODE_ENV, ENVIRONMENTS,
+	SENTRY_DSN,
+	SC_DOMAIN,
+	NODE_ENV,
+	ENVIRONMENTS,
+	SENTRY_SAMPLE_RATE,
```

Comment:
This is not defined in globals, isn't it not? Use configuration.get directly in line 114 and 118 below.